deploy/vagrant/bin/spot_request.py

```diff
@@ -82,7 +82,7 @@ def get_host(tag):
 def request_id_to_tag(requests):
     ret = {}
     for i, rid in enumerate([r.id for r in requests]):
-        host = "TachyonMaster" if i == 0 else "TachyonWorker{}".format(i)
+        host = u'TachyonMaster' if i == 0 else "TachyonWorker{}".format(i)
```

Comment:
`for the lookup to work`, I'm wondering for lookup of what? I thought this is just the tag name sent to aws?